src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -377,15 +360,12 @@ public void run() {
     void reloadBundle(final Key key) {
         log.info("Reloading resource bundle for {}", key);
         if (!this.preloadBundles) {
-            // remove bundle from cache
-            resourceBundleCache.remove(key);
-            // unregister bundle
-            unregisterResourceBundle(key);
+            resourceBundleRegistry.unregisterRB(key);
```

Comment:
Same as above, I'd prefer `unregisterResourceBundle(key)`.

Reply:
Ok for me.